README.md

```diff
@@ -9,13 +9,13 @@ Author's Note:
 >
 >As much as I would like to say this guide is accurate (yes, disclaimer time) this is by no means an "official" guide and is written solely from the perspective of students like myself with sources to official documentation. You should make sure to consult with the Undergrad Office or your college registrar to confirm your understanding of these topics and to verify your plans.  
 >
->This is also a **WORK IN PROGRESS**, myself and others will be adding and editing content as we get the time to.
+>This is also **WORK IN PROGRESS**, myself and others will be adding and editing content as we get the time to.
```

Comment:
```suggestion
>This is also a **WORK IN PROGRESS**, myself and others will be adding and editing content as we get the time to.
```
I would keep the "a" it seems weird otherwise

Reply:
done